Assets/Scripts/Player/Inventory/InventoryDisplay.cs

```diff
@@ -0,0 +1,65 @@
+using Examen.Items;
+using FishNet.Object;
+using UnityEngine;
+
+namespace Examen.Inventory
+{
+    public class InventoryDisplay : NetworkBehaviour
+    {
+        [SerializeField] private InventoryDisplayItem _displayItem;
+        [SerializeField] private Transform[] _contents;
+
+        private InventoryPackage _currentPackage;
+
+        private void OnEnable() 
+        {
+            InventorySystem.Instance.OnItemsChanged += UpdateDisplay;
+        }
```

Comment:
I assume you did this because otherwise the line would be too long, but you can make it like this:
```suggestion
        private void OnEnable() 
        => InventorySystem.Instance.OnItemsChanged += UpdateDisplay;
```
or this:
```suggestion
        private void OnEnable() =>
           InventorySystem.Instance.OnItemsChanged += UpdateDisplay;
```